src/cata_tiles.h

```diff
@@ -217,6 +235,12 @@ class cata_tiles
 
         /** Surface/Sprite rotation specifics */
         SDL_Surface *create_tile_surface();
+        SDL_Surface *create_tile_surface(int w, int h);
+        void convert_surface_to_grayscale(SDL_Surface *surf);
```

Comment:
As none of these new methods are called from outside cata_tiles, they should be private instead of protected.